Hast.Abstractions/Hast.Vitis.Abstractions/Models/AzureAttestationConfiguration.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Hast.Layer;
+using System;
+
+namespace Hast.Vitis.Abstractions.Models
+{
+    public class AzureAttestationConfiguration : AzureStorageConfiguration
+    {
+        /// <summary>
+        /// Gets or sets the Azure function URL for blob attestation submission.
+        /// </summary>
+        public Uri StartFunctionUrl { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Azure function URL for polling the status of the attestation process.
+        /// </summary>
+        public Uri PollFunctionUrl { get; set; }
+
+        /// <summary>
+        /// Gets or sets the ID of the user hosting the storage blob. You can retrieve it using:
+        /// <code>
+        /// az account show --output tsv --query id
+        /// </code>
+        /// </summary>
+        public string ClientSubscriptionId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the ID of the tenant hosting the storage blob. You can retrieve it using:
+        /// <code>
+        /// az account show --output tsv --query tenantId
+        /// </code>
+        /// </summary>
+        public string ClientTenantId { get; set; }
+
+        public AzureAttestationConfiguration SetupAndVerify()
+        {
+            VerifyConfiguration();
+            Environment.SetEnvironmentVariable("AZURE_STORAGE_CONNECTION_STRING", GenerateStorageConnectionString());

```

Comment:
Ezt mihez kell beállítani? Dokumentáld ide.

Reply:
[Itt olvastam](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-dotnet#configure-your-storage-connection-string), de jobban átgondolva mégse kell.